es-ES.js

```diff
@@ -1,150 +1,150 @@
 export default {
 	categories: {
 		gameOne: {
-			title: 'Game One',
-			desc: 'Create a game with an existing engine of your choice, e.g. GameMaker or Unity.',
+			title: 'Primer juego',
+			desc: 'Crea un juego con una motor existente a tu elección, por ejemplo: GameMaker o Unity.',
 		},
 		gameTwo: {
-			title: 'Game Two',
-			desc: 'Create a game from scratch with your engine and compiled with vitaSDK or dolceSDK.',
+			title: 'Segundo juego',
+			desc: 'Crea un juego desde 0 con tu motor y compliado con vitaSDK o dolceSDK.',
 		},
 		utility: {
-			title: 'Utility',
-			desc: 'Create a homebrew or a plugin.',
+			title: 'Herramienta',
+			desc: 'Crea un homebrew o plugin.',
 		},
 		port: {
 			title: 'Port',
-			desc: 'Port an application or a game from another system, or an emulator.',
+			desc: 'Porta una aplicación, juego o emulador de otro sistema.',
 		},
 	},
 
 	rules: {
 		submissionsLimit:
-			'Developers can submit as many submissions as they want but can only win one prize per category.',
+			'Los desarrolladores pueden presentar todo lo que quieran pero soo podrán ganar un premio por categoría.',
 		previouslySubmitted:
-			'For games, if they were previously released on another platform like PS4 or Switch, even if it is yours, it must be submitted in the port category. Utility from other homebrew scenes shall remain in the utility category though',
+			'Para los juegos, si anteriormente habían salido en otra plataforma, por ejemplo en PS4 o Switch, aunque sea tu juego, deben ser enviados a la categoría de ports. Las herramientas de homebrew de otras escenas deben ser enviadas a la categoría herramientas',
 		releasedBefore:
-			'If the game you want to submit was already released before on the PSVita, it must have been at the state of prototype and being improved to make it a complete game. For utility, the changes should be consequent to the homebrew/plugins which required lot of work.',
+			'Si el juego que quieres enviar ya había salido en PS Vita, debe quedar en estado de prototipo y tener mejoras respecto al anterior para que sea un juego completo. Para las herramientas, los cambios deben ser consecuentes ya que los homebrew/plugins requieren mucho trabajo.',
```

Comment:
'debe quedar en estado de prototipo' is incorrect, it should be 'tuvo que ser en estado de prototipo'

'Para las herramientas, los cambios deben ser consecuentes ya que los homebrew/plugins requieren mucho trabajo.' doesn't make sense, even though the original text doesn't make much sense either lol. 
Try with 'Para los plugins o herramientas, los cambios deben ser significativos y consecuentes respecto a las versiones anteriores.'
This does change the meaning of the original phrase to "for plugins and tools, the changes must be significative and consecuent to the previous versions" tho.